com.unity.cinemachine/Samples~/3D Samples/RunningRace.unity

```diff
@@ -1092,6 +1208,149 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: dab5c7d4c32e743048dfca98e2d5914f, type: 3}
   m_Name: 
   m_EditorClassIdentifier: 
+--- !u!1 &926545930
+GameObject:
```

Comment:
Why the word "Active" in the buttons?  
![image](https://user-images.githubusercontent.com/6424658/231187499-467ca2fb-156c-4b7f-8e71-c3a985cb6f77.png)

Reply:
We don't need Activate